### PR TITLE
`hotshot/rc-0.5.81`

### PR DESCRIPTION
**Pull request by rob-maron**

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -17,8 +17,6 @@ jobs:
     runs-on: ubuntu-latest
 
     env:
-      RUSTFLAGS: '--cfg async_executor_impl="async-std" --cfg async_channel_impl="async-std"'
```
Comment (QuentinI) [inline .github/workflows/lint.yml:20]: Our savior has arrived 🎉

**Review by QuentinI — COMMENTED** [review]
Is there a reason we want to move to tokio channels instead of async_broadcast ones?

**Comment by rob-maron** [comment]
> Is there a reason we want to move to tokio channels instead of async_broadcast ones?

My goal was to remove the `async-compatibility-layer` channels and replace them with Tokio ones. Async-broadcast should still be pulled in everywhere we were using it before -> did I miss one?

The reason for removing `async-compat` ones was because that still transitively requires the RUSTFLAGS 😅

**Comment by QuentinI** [comment]
> My goal was to remove the async-compatibility-layer channels and replace them with Tokio ones. Async-broadcast should still be pulled in everywhere we were using it before -> did I miss one?

No, you're right, sorry, I misread some of the lines as replacing async_broadcast imports

**Review by QuentinI — APPROVED** [review]

**Pull request merged**